build/dependencies.props

```diff
@@ -26,6 +26,8 @@
   <PropertyGroup Label="Package Versions: Pinned">
     <!-- Packages that came from aspnet/Extensions. -->
     <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>2.1.1</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
+    <!-- Adding MVC here until it is moved to project ref -->
+    <MicrosoftAspNetCoreMvcPackageVersion>2.1.1</MicrosoftAspNetCoreMvcPackageVersion>
```

Comment:
Can we do MVC first? Doing these out of order will break the master branch.

@ajaybhargavb - are you looking into converting MVC to project ref?

Reply:
I haven't started. I can start on this today.